cm_sdk/models/__init__.py

```diff
@@ -0,0 +1,266 @@
+from enum import Enum
+import json
+from json import JSONEncoder
+import re
+import logging
+
+__author__ = 'e0d'
+
+
+class StringUtil(object):
```

Comment:
You should move these out of `__init__.py` and into a child package. You can expose them via `__all__` if you so choose.